.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -16,11 +16,13 @@ jobs:
     CONDA_BLD_PATH: D:\\bld\\
     MINIFORGE_HOME: D:\Miniforge
     UPLOAD_TEMP: D:\\tmp
+    SET_PAGEFILE: 'True'
 
   steps:
 
     - script: |
         call ".scripts\run_win_build.bat"
+        set "CMAKE_ARGS=%CMAKE_ARGS% -DCMAKE_INSTALL_LIBDIR:STRING=%LIBRARY_LIB% -DCMAKE_INSTALL_PREFIX:STRING=%LIBRARY_PREFIX% -DCMAKE_PREFIX_PATH:STRING=%LIBRARY_PREFIX% -DBUILD_SHARED_LIBS:BOOL=ON -DANTS_SUPERBUILD:BOOL=OFF -DUSE_SYSTEM_ITK:BOOL=ON -DCMAKE_BUILD_TYPE:STRING=Release"
```

Comment:
Why are you doing this? This should have no effect. Better rerender cleanly again.